src/plugins/ovp/components/kaltura-entry-fields/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Card, CardBody, CardHeader } from 'reactstrap';
```

Comment:
@rsumilang same here, no need for first class component.  fields names are invalid.  remove proptypes